src/viewers/VivViewer.js

```diff
@@ -98,27 +93,6 @@ export default class VivViewer extends PureComponent {
       });
       return { viewStates };
     }
-    if (
-      views.some(
-        view =>
-          view.initialViewState.target !==
-            prevState.initialViewStates[view.id].target ||
-          view.initialViewState.zoom !==
-            prevState.initialViewStates[view.id].zoom
-      )
-    ) {
-      const initialViewStates = {};
-      const viewStates = {};
-      views.forEach(view => {
-        viewStates[view.id] = view.filterViewState({
-          viewState: view.initialViewState
-        });
-        initialViewStates[view.id] = view.filterViewState({
-          viewState: view.initialViewState
-        });
-      });
-      return { initialViewStates, viewStates };
-    }
```

Comment:
nice to get rid of all this weird diffing between initialviewstate and viewstate; it's just single state now. good choice.